Project.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+using namespace std;
+
+class User{
+    public:
+    string Username;
+    string Email;
+    User(string username,string email){
+        Username = username;
+        Email = email;
+    }
+    void print(){
+        std::cout<<"Username - "<<Username<<endl;
+        std::cout<<"Email - "<<Email<<endl;
+    }
+    void changeUsername(string newUsername){
+        Username = newUsername;
+    }
+};
+
+class Note{
+    public:
+    string TopicName;
+    string NoteContent;
+    Note(string topic, string noteContent){ 
+        TopicName = topic;
+        NoteContent = noteContent; 
+    }
+    void print(){
+        std::cout<<"Topic - "<<TopicName<<endl;
+        std::cout<<"NoteContent - "<<NoteContent<<endl;
+    }
+    void changeTopic(string newTopic){
+        TopicName = newTopic;
+    }
+};
+int main(){
+    User user1("Piyush","piyushb@gmail.com");
```

Comment:
Consider using smart pointers instead of raw pointers to manage the lifecycle of objects more safely and efficiently. [medium]